client/grpc/grpc.go

```diff
@@ -86,9 +87,15 @@ func unaryInterceptor(target string) grpc.UnaryClientInterceptor {
 		invokeDuration := time.Since(invokeTime)
 
 		rpcStatus, _ := status.FromError(err) // codes.OK if err == nil, codes.Unknown if !ok
-		rpcDurationMetrics.
-			WithLabelValues(unary, target, method, rpcStatus.Code().String()).
-			Observe(invokeDuration.Seconds())
+
+		durationHistogram := rpcDurationMetrics.WithLabelValues(unary, target, method, rpcStatus.Code().String())
+		if sctx, ok := span.Context().(jaeger.SpanContext); ok {
+			durationHistogram.(prometheus.ExemplarObserver).ObserveWithExemplar(
+				invokeDuration.Seconds(), prometheus.Labels{"traceID": sctx.TraceID().String()},
```

Comment:
"traceID" seems to be a good candidate as a constant in the trace package.

Reply:
created a constant as suggested